src/views/Viewpoint/DetailViewpoint/SidebarTabsContent/InformationTab/Minimap/Minimap.js

```diff
@@ -0,0 +1,98 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { CONTROLS_BOTTOM_RIGHT } from '@terralego/core/modules/Map/Map';
+import InteractiveMap, { CONTROL_BACKGROUND_STYLES } from '@terralego/core/modules/Map/InteractiveMap';
+import { useTranslation } from 'react-i18next';
+
+import { addCustomIcon } from '../../../../../../components/Visualizer/Visualizer';
+import MapErrorConfiguration from '../../../../../../components/MapErrorConfiguration';
+
+import './Minimap.scss';
+
+const Minimap = ({
+  configMap: { zoom, ...configMap },
+  coordinates,
+}) => {
+  const {
+    i18n: {
+      getResourceBundle,
+      language,
+      store: { options: { fallbackLng } },
+    },
+    t,
+  } = useTranslation();
+
+  const layers = [{
+    id: 'viewpoint',
+    type: 'symbol',
+    source: {
+      type: 'geojson',
+      data: {
+        type: 'Point',
+        coordinates,
+      },
+    },
+    layout: {
+      'icon-image': 'marker-poi',
+      'icon-size': 0.5,
+      'icon-anchor': 'bottom',
+      'icon-allow-overlap': true,
+    },
+    weight: 850,
+  }];
+
+  const props = useMemo(() => ({
+    ...configMap,
+    customStyle: { layers },
+    zoom: zoom + 1,
+    center: coordinates,
+    onMapInit: addCustomIcon,
+    controls: [{
+      control: CONTROL_BACKGROUND_STYLES,
+      position: CONTROLS_BOTTOM_RIGHT,
+    }],
+  }), [configMap, coordinates, layers, zoom]);
+
+
+  if (!configMap.accessToken || !configMap.backgroundStyle) {
+    return (
+      <MapErrorConfiguration />
+    );
+  }
+
+  const { terralego: { map: locale } } = getResourceBundle(language.split('-')[0]) || getResourceBundle(fallbackLng[0]);
```

Comment:
The bundle loading shouldn't be at higher level ?

Reply:
As seen together, i just removed this piece of code 👍